src/init.py

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/python
+
+import argparse
+import os
+
+from kubernetes import client, config
+from kubernetes.client import ApiClient, Configuration
+
+from openshift.dynamic import DynamicClient
+
+parser = argparse.ArgumentParser(description="Options to Program")
+parser.add_argument('-n',default="openshift-machine-api",dest="namespace",help="Namespace to use")
```

Comment:
In python we should have a space after each coma, if we could fix all these :) Threw me off a little when I was reading it.

Reply:
Ooh, I thought I got all of those. Will fix them.